.github/workflows/build.yml

```
@@ -17,10 +17,9 @@ jobs:
         strategy:
             fail-fast: false
             matrix:
-                php: ["8.0", "8.1"]
+                php: ["8.0", "8.1", "8.2"]
                 postgres: [""]
-                symfony: ["^5.4", "^6.0"]
-
+                symfony: ["^5.4", "^6.0", "^6.1", "^6.2"]
```

```suggestion
                symfony: ["^5.4", "~6.0.0", "~6.1.0", "~6.2.0"]
```
Take a look at the builds, everything with PHP 8.1+ gets Symfony 6.2 when using a caret